Default.sublime-commands

```diff
@@ -52,8 +53,11 @@
         "command": "lsp_format_document_range",
     },
     {
-        "caption": "LSP: Restart Servers",
-        "command": "lsp_restart_client",
+        "caption": "LSP: Restart Server",
+        "command": "lsp_restart_server",
+        "args": {
+            "show_quick_panel": true
+        }
```

Comment:
If the `config_name` parameter is `None`, wouldn't that already imply we have to show a quick panel? I think you can remove the `show_quick_panel` parameter.